modules/core/04-channel/types/packet.go

```diff
@@ -58,6 +123,28 @@ func NewPacket(
 		DestinationChannel: destinationChannel,
 		TimeoutHeight:      timeoutHeight,
 		TimeoutTimestamp:   timeoutTimestamp,
+		ProtocolVersion:    IBC_VERSION_CLASSIC,
```

Comment:
Why is protocol version contained in the ack commitment but not the packet commitment?

Reply:
protocol version is not in ack commitment?

Reply:
Ah, sorry